components/google-cloud/google_cloud_pipeline_components/experimental/custom_job/custom_job.py

```diff
@@ -42,6 +42,9 @@ def run_as_vertex_ai_custom_job(
     service_account: Optional[str] = None,
     network: Optional[str] = None,
     worker_pool_specs: Optional[List[Mapping[str, Any]]] = None,
+    encryption_spec_key_name: Optional[str] = None,
```

Comment:
this might not work as user expect. such configs could be pipeline runtime parameters, and putting them here will not allow user to refer to the runtime parameters.
same thing for the service_account/network above.

Reply:
talked offline please ignore this comment. i think this should work.

Reply:
Acknowledged.